src/main/resources/view/MainWindow.fxml

```diff
@@ -31,26 +30,28 @@
           <Menu mnemonicParsing="false" text="Help">
             <MenuItem fx:id="helpMenuItem" mnemonicParsing="false" onAction="#handleHelp" text="Help" />
           </Menu>
+          <Menu mnemonicParsing="false" text="Job">
+            <MenuItem fx:id="jobMenuItem" mnemonicParsing="false" onAction="#handleJob" text="Job" />
```

Comment:
Using "job" as the label might be a bit miss leading, maybe can use smth like "Job Schedule", "Job Timetable" etc